File: Free Html To Pdf Converter for ASP.NET MVC/[C#]-Free Html To Pdf Converter for ASP.NET MVC/C#/SelectPdf.Samples/Controllers/ViewHttpHeadersController.cs
using System.Web.Mvc;

namespace SelectPdf.Samples.Controllers
{
    public class ViewHttpHeadersController : Controller
    {
        // GET: ViewHttpHeaders
        public ActionResult Index()
        {
            string headers = string.Empty;
            foreach (string name in Request.Headers)
            {
                string value = Request.Headers[name];
                headers += "<br/>" + name + ": " + value;
            }

            ViewData.Add("HeadersValue", headers);
            
            return View();
        }
    }
}

File: Free Html To Pdf Converter for ASP.NET MVC/[VB]-Free Html To Pdf Converter for ASP.NET MVC/VB/SelectPdf.Samples/Controllers/ViewHttpHeadersController.vb
Namespace Controllers
    Public Class ViewHttpHeadersController
        Inherits Controller

        ' GET: ViewHttpHeaders
        Public Function Index() As ActionResult
            Dim headers As String = String.Empty
            For Each name As String In Request.Headers
                Dim value As String = Request.Headers(name)
                headers += "<br/>" & name & ": " & value
            Next

            ViewData.Add("HeadersValue", headers)

            Return View()
        End Function
    End Class
End Namespace